### PR TITLE
Second round replacing #ifndef guards by #pragma once include guards

### PR DESCRIPTION
**Pull request by SunBlack**
Continues #429/#2617. Last time I searched for `#include PCL`, now I searched just for `#ifndef`. I just skipped `3rdparty` directory (opennurbs/poisson4), because I'm not sure how you handle 3rdParty libraries.

There could be a compile issue, because some macros were multiple times used:
https://github.com/PointCloudLibrary/pcl/blob/5cef903b3be806105cfa9f8600e8bda9d3650a6b/doc/tutorials/content/sources/iros2011/include/object_recognition.h#L1-L2
https://github.com/PointCloudLibrary/pcl/blob/5cef903b3be806105cfa9f8600e8bda9d3650a6b/doc/tutorials/content/sources/iros2011/include/solution/object_recognition.h#L1-L2

https://github.com/PointCloudLibrary/pcl/blob/5cef903b3be806105cfa9f8600e8bda9d3650a6b/doc/tutorials/content/sources/iros2011/include/surface.h#L1-L2
https://github.com/PointCloudLibrary/pcl/blob/5cef903b3be806105cfa9f8600e8bda9d3650a6b/doc/tutorials/content/sources/iros2011/include/solution/surface.h#L1-L2

I didn't checked if we can remove solution directory or not, because it seems to be just a copy of parent directory.

### CODE REVIEW

**Comment by taketwo** [comment]
Please double-check if the tutorial files that you modified are referenced in tutorial text. If so, code block includes should be adjusted. (See for example this commit: https://github.com/PointCloudLibrary/pcl/pull/2696/commits/bed8e54106cad98b62ccd35a626db982abffb16c.)

**Comment by SunBlack** [comment]
Searched for `literalinclude.*\.h` and fixed all occurrences.

File: doc/tutorials/content/qt_colorize_cloud.rst [hunk 1]
```diff
@@ -60,13 +60,13 @@ pclviewer.h
...
    :language: cpp
-   :lines: 42-57
+   :lines: 41-56
 
 These are the public slots triggered by the buttons in the UI.
 
 .. literalinclude:: sources/qt_colorize_cloud/pclviewer.h
    :language: cpp
-   :lines: 59-86
+   :lines: 89-85
```
Comment (taketwo) [inline doc/tutorials/content/qt_colorize_cloud.rst]: 58?

Comment (SunBlack) [reply doc/tutorials/content/qt_colorize_cloud.rst]: Ups xD

**Pull request merged**